pkg/plugins/backendplugin/manager.go

```diff
@@ -224,9 +227,15 @@ func (m *manager) CallResource(config PluginConfig, c *models.ReqContext, path s
 		User:    c.SignedInUser,
 	}
 
+	status := "ok"
```

Comment:
In `// Expected that headers and status are only part of first stream` below you should expect status code to be available. When ` err == io.EOF` stream is finished and you can measure duration. But maybe need a specific metric to capture status code (if we want that).

Reply:
Made some changes for this.